indexing-service/src/main/java/org/apache/druid/indexing/overlord/supervisor/SupervisorResource.java

```diff
@@ -143,6 +148,19 @@ public Response specPost(final SupervisorSpec spec, @Context final HttpServletRe
           }
 
           manager.createOrUpdateAndStartSupervisor(spec);
+
+          final String auditPayload
```

Comment:
do we log the update timestamp in the audit event that can be used to match the user activity with the entry in the supervisor history?

Reply:
An audit entry does contain the audit time, but it is currently not being logged.

The log message is of the format
```
User[%s], identity[%s], IP[%s] performed action of type[%s] on key[%s] with comment[%s], request[%s], payload[%s].
```

The audit time had been left out of the log message on purpose as it is typically the same as the timestamp of the log message itself.